### ISSUE TITLE
Google PageSpeed Unhandled error response on desktop but lighthouse on dev mode is working

### ISSUE THREAD

**Issue by ninesixfiveseven** [GoogleChrome/lighthouse#15736]
My website includes React Three Fiber, ACF Plugin, WordPress, and Cloudflare. Other pages without React Three Fiber can use pagespeed, but the homepage with React Three Fiber cannot. In desktop mode, it shows the error "generic::internal: APP::1: Unhandled error response: Execution was terminated." However, when I use Lighthouse in dev mode, the speed test works fine.

**Comment by ninesixfiveseven** [comment]
website is not slowly, in lighthouse performance is around 80 on desktop, and another page 90++

**Comment by ninesixfiveseven** [comment]
> Can you please provide the URL of the site this is happening on

This is [PageSpeed Insights ](https://pagespeed.web.dev/analysis/https-invisibleink-asia/3uc2c63ymk?form_factor=mobile) and this is my [site](https://invisibleink.asia/)

**Comment by ninesixfiveseven** [comment]
In mobile score on pagespeed insight that lower than lighthouse, and pagespeed insight can not analyze desktop mode

**Comment by adamraine** [comment]
Thanks I can actually reproduce this outside of PSI using Lighthouse normally. Lighthouse takes ~2.5 minutes to finish on my local machine but looks like PSI times out after ~2 minutes. Looking at the output, Lighthouse first times out waiting for the page to load and then takes a very long to finish collecting several artifacts (`RootCauses`, `Accessibility`, etc):

```
$ lighthouse https://invisibleink.asia/ --preset=desktop --chrome-flags="--headless=old"
LH:ChromeLauncher Waiting for browser. +0ms
LH:ChromeLauncher Waiting for browser... +0ms
LH:ChromeLauncher Waiting for browser..... +504ms
LH:ChromeLauncher Waiting for browser.....✓ +1ms
LH:status Connecting to browser +648ms
LH:status Navigating to about:blank +6ms
LH:status Benchmarking machine +6ms
LH:status Preparing target for navigation mode +1s
LH:status Cleaning origin data +14ms
LH:status Cleaning browser cache +85ms
LH:status Preparing network conditions +9ms
LH:status Navigating to https://invisibleink.asia/ +115ms
LH:waitFor:warn Timed out waiting for page load. Checking if page is hung... +45s
LH:status Getting artifact: DevtoolsLog +9s
LH:status Getting artifact: Trace +0ms
LH:status Getting artifact: DevtoolsLog +4ms
LH:status Getting artifact: Trace +0ms
LH:status Getting artifact: RootCauses +0ms
LH:status Getting artifact: Accessibility +47s
LH:status Getting artifact: AnchorElements +3s
LH:status Getting artifact: ConsoleMessages +4s
LH:status Getting artifact: CSSUsage +0ms
LH:status Getting artifact: Doctype +4s
LH:status Getting artifact: DOMStats +1ms
LH:status Getting artifact: EmbeddedContent +5ms
LH:status Getting artifact: FontSize +486ms
LH:status Getting artifact: Inputs +2s
LH:status Getting artifact: GlobalListeners +548ms
LH:status Getting artifact: ImageElements +950ms
LH:ImageElements:warn Reached gathering budget of 5s. Skipped extra details for 38/46 +6s
LH:status Getting artifact: InstallabilityErrors +456ms
LH:status Get webapp installability errors +0ms
LH:status Getting artifact: InspectorIssues +2ms
LH:status Getting artifact: JsUsage +0ms
LH:status Getting artifact: LinkElements +0ms
LH:status Getting artifact: MainDocumentContent +468ms
LH:status Getting artifact: MetaElements +483ms
LH:status Getting artifact: NetworkUserAgent +480ms
LH:status Getting artifact: OptimizedImages +0ms
LH:status Getting artifact: ResponseCompression +1ms
LH:status Getting artifact: RobotsTxt +0ms
LH:status Getting artifact: Scripts +573ms
LH:status Getting artifact: SourceMaps +0ms
LH:status Getting artifact: Stacks +2s
LH:status Collect stacks +0ms
LH:status Getting artifact: TagsBlockingFirstPaint +797ms
LH:status Getting artifact: TapTargets +493ms
LH:status Getting artifact: TraceElements +2s
LH:status Getting artifact: ViewportDimensions +7s
LH:status Getting artifact: WebAppManifest +1ms
LH:status Get webapp manifest +0ms
LH:status Getting artifact: devtoolsLogs +1ms
LH:status Getting artifact: traces +0ms
LH:status Getting artifact: FullPageScreenshot +0ms
LH:status Getting artifact: BFCacheFailures +7s
LH:status Analyzing and running audits... +623ms
LH:status Auditing: Uses HTTPS +2ms
LH:status Auditing: Has a `<meta name="viewport">` tag with `width` or `initial-scale` +3ms
LH:status Auditing: First Contentful Paint +4ms
LH:status Auditing: Largest Contentful Paint +166ms
LH:status Auditing: First Meaningful Paint +16ms
LH:status Auditing: Speed Index +8ms
LH:status Auditing: Screenshot Thumbnails +530ms
LH:status Auditing: Final Screenshot +0ms
LH:status Auditing: Total Blocking Time +7ms
LH:status Auditing: Max Potential First Input Delay +31ms
LH:status Auditing: Cumulative Layout Shift +17ms
LH:status Auditing: No browser errors logged to the console +1ms
LH:status Auditing: Initial server response time was short +2ms
LH:status Auditing: Time to Interactive +2ms
LH:status Auditing: User Timing marks and measures +2ms
LH:status Auditing: Avoid chaining critical requests +12ms
LH:status Auditing: Avoid multiple page redirects +5ms
LH:status Auditing: Web app manifest and service worker meet the installability requirements +13ms
LH:status Auditing: Configured for a custom splash screen +4ms
LH:status Auditing: Sets a theme color for the address bar. +2ms
LH:status Auditing: Manifest has a maskable icon +2ms
LH:status Auditing: Content is sized correctly for the viewport +2ms
LH:status Auditing: Displays images with correct aspect ratio +1ms
LH:status Auditing: Serves images with appropriate resolution +2ms
LH:status Auditing: Fonts with `font-display: optional` are preloaded +2ms
LH:status Auditing: Avoids deprecated APIs +2ms
LH:status Auditing: Avoids third-party cookies +2ms
LH:status Auditing: Minimizes main-thread work +2ms
LH:status Auditing: JavaScript execution time +79ms
LH:status Auditing: Preload key requests +71ms
LH:status Auditing: Preconnect to required origins +1ms
LH:status Auditing: All text remains visible during webfont loads +5ms
LH:status Auditing: Diagnostics +8ms
LH:status Auditing: Network Requests +2ms
LH:status Auditing: Network Round Trip Times +7ms
LH:status Auditing: Server Backend Latencies +2ms
LH:status Auditing: Tasks +1ms
LH:status Auditing: Metrics +1ms
LH:status Auditing: Performance budget +9ms
LH:status Auditing: Timing budget +2ms
LH:status Auditing: Resources Summary +1ms
LH:status Auditing: Minimize third-party usage +1ms
LH:status Auditing: Lazy load third-party resources with facades +21ms
LH:status Auditing: Largest Contentful Paint element +18ms
LH:status Auditing: Largest Contentful Paint image was not lazily loaded +2ms
LH:status Auditing: Avoid large layout shifts +2ms
LH:status Auditing: Avoid large layout shifts +4ms
LH:status Auditing: Avoid long main-thread tasks +3ms
LH:status Auditing: Avoids `unload` event listeners +11ms
LH:status Auditing: Avoid non-composited animations +2ms
LH:status Auditing: Image elements have explicit `width` and `height` +1ms
LH:status Auditing: Page has valid source maps +2ms
LH:status Auditing: Preload Largest Contentful Paint image +3ms
LH:status Auditing: Ensure CSP is effective against XSS attacks +1ms
LH:status Auditing: Script Treemap Data +1ms
LH:status Auditing: Site works cross-browser +35ms
LH:status Auditing: Page transitions don't feel like they block on the network +1ms
LH:status Auditing: Each page has a URL +1ms
LH:status Auditing: `[accesskey]` values are unique +1ms
LH:status Auditing: `[aria-*]` attributes match their roles +2ms
LH:status Auditing: Values assigned to `role=""` are valid ARIA roles. +4ms
LH:status Auditing: `button`, `link`, and `menuitem` elements have accessible names +4ms
LH:status Auditing: Elements with `role="dialog"` or `role="alertdialog"` have accessible names. +4ms
LH:status Auditing: `[aria-hidden="true"]` is not present on the document `<body>` +2ms
LH:status Auditing: `[aria-hidden="true"]` elements do not contain focusable descendents +4ms
LH:status Auditing: ARIA input fields have accessible names +4ms
LH:status Auditing: ARIA `meter` elements have accessible names +2ms
LH:status Auditing: ARIA `progressbar` elements have accessible names +1ms
LH:status Auditing: `[role]`s have all required `[aria-*]` attributes +1ms
LH:status Auditing: Elements with an ARIA `[role]` that require children to contain a specific `[role]` have all required children. +4ms
LH:status Auditing: `[role]`s are contained by their required parent element +1ms
LH:status Auditing: `[role]` values are valid +2ms
LH:status Auditing: Elements with the `role=text` attribute do not have focusable descendents. +3ms
LH:status Auditing: ARIA toggle fields have accessible names +6ms
LH:status Auditing: ARIA `tooltip` elements have accessible names +2ms
LH:status Auditing: ARIA `treeitem` elements have accessible names +2ms
LH:status Auditing: `[aria-*]` attributes have valid values +2ms
LH:status Auditing: `[aria-*]` attributes are valid and not misspelled +4ms
LH:status Auditing: Buttons have an accessible name +4ms
LH:status Auditing: The page contains a heading, skip link, or landmark region +4ms
LH:status Auditing: Background and foreground colors have a sufficient contrast ratio +4ms
LH:status Auditing: `<dl>`'s contain only properly-ordered `<dt>` and `<dd>` groups, `<script>`, `<template>` or `<div>` elements. +4ms
LH:status Auditing: Definition list items are wrapped in `<dl>` elements +2ms
LH:status Auditing: Document has a `<title>` element +9ms
LH:status Auditing: `[id]` attributes on active, focusable elements are unique +4ms
LH:status Auditing: ARIA IDs are unique +3ms
LH:status Auditing: All heading elements contain content. +4ms
LH:status Auditing: No form fields have multiple labels +4ms
LH:status Auditing: `<frame>` or `<iframe>` elements have a title +2ms
LH:status Auditing: Heading elements appear in a sequentially-descending order +2ms
LH:status Auditing: `<html>` element has a `[lang]` attribute +4ms
LH:status Auditing: `<html>` element has a valid value for its `[lang]` attribute +4ms
LH:status Auditing: `<html>` element has an `[xml:lang]` attribute with the same base language as the `[lang]` attribute. +4ms
LH:status Auditing: Identical links have the same purpose. +9ms
LH:status Auditing: Image elements have `[alt]` attributes +4ms
LH:status Auditing: Image elements do not have `[alt]` attributes that are redundant text. +4ms
LH:status Auditing: Input buttons have discernible text. +4ms
LH:status Auditing: `<input type="image">` elements have `[alt]` text +3ms
LH:status Auditing: Elements with visible text labels have matching accessible names. +2ms
LH:status Auditing: Form elements have associated labels +4ms
LH:status Auditing: Document has a main landmark. +3ms
LH:status Auditing: Links have a discernible name +3ms
LH:status Auditing: Links are distinguishable without relying on color. +9ms
LH:status Auditing: Lists contain only `<li>` elements and script supporting elements (`<script>` and `<template>`). +3ms
LH:status Auditing: List items (`<li>`) are contained within `<ul>`, `<ol>` or `<menu>` parent elements +4ms
LH:status Auditing: The document does not use `<meta http-equiv="refresh">` +4ms
LH:status Auditing: `[user-scalable="no"]` is not used in the `<meta name="viewport">` element and the `[maximum-scale]` attribute is not less than 5. +3ms
LH:status Auditing: `<object>` elements have alternate text +4ms
LH:status Auditing: Select elements have associated label elements. +2ms
LH:status Auditing: Skip links are focusable. +3ms
LH:status Auditing: No element has a `[tabindex]` value greater than 0 +3ms
LH:status Auditing: Tables have different content in the summary attribute and `<caption>`. +3ms
LH:status Auditing: Tables use `<caption>` instead of cells with the `[colspan]` attribute to indicate a caption. +8ms
LH:status Auditing: Touch targets have sufficient size and spacing. +3ms
LH:status Auditing: `<td>` elements in a large `<table>` have one or more table headers. +4ms
LH:status Auditing: Cells in a `<table>` element that use the `[headers]` attribute refer to table cells within the same table. +3ms
LH:status Auditing: `<th>` elements and elements with `[role="columnheader"/"rowheader"]` have data cells they describe. +4ms
LH:status Auditing: `[lang]` attributes have a valid value +3ms
LH:status Auditing: `<video>` elements contain a `<track>` element with `[kind="captions"]` +4ms
LH:status Auditing: Custom controls have associated labels +3ms
LH:status Auditing: Custom controls have ARIA roles +1ms
LH:status Auditing: User focus is not accidentally trapped in a region +0ms
LH:status Auditing: Interactive controls are keyboard focusable +0ms
LH:status Auditing: Interactive elements indicate their purpose and state +0ms
LH:status Auditing: The page has a logical tab order +0ms
LH:status Auditing: The user's focus is directed to new content added to the page +0ms
LH:status Auditing: Offscreen content is hidden from assistive technology +0ms
LH:status Auditing: HTML5 landmark elements are used to improve navigation +0ms
LH:status Auditing: Visual order on the page follows DOM order +0ms
LH:status Auditing: Uses efficient cache policy on static assets +1ms
LH:status Auditing: Avoids enormous network payloads +10ms
LH:status Auditing: Defer offscreen images +4ms
LH:status Auditing: Eliminate render-blocking resources +22ms
LH:status Auditing: Minify CSS +152ms
LH:status Auditing: Minify JavaScript +146ms
LH:status Auditing: Reduce unused CSS +190ms
LH:status Auditing: Reduce unused JavaScript +7ms
LH:status Auditing: Serve images in next-gen formats +8ms
LH:status Auditing: Efficiently encode images +7ms
LH:status Auditing: Enable text compression +8ms
LH:status Auditing: Properly size images +7ms
LH:status Auditing: Use video formats for animated content +8ms
LH:status Auditing: Remove duplicate modules in JavaScript bundles +8ms
LH:status Auditing: Avoid serving legacy JavaScript to modern browsers +8ms
LH:status Auditing: Page has the HTML doctype +167ms
LH:status Auditing: Properly defines charset +2ms
LH:status Auditing: Avoids an excessive DOM size +1ms
LH:status Auditing: Avoids requesting the geolocation permission on page load +6ms
LH:status Auditing: No issues in the `Issues` panel in Chrome Devtools +1ms
LH:status Auditing: Avoids `document.write()` +2ms
LH:status Auditing: Detected JavaScript libraries +1ms
LH:status Auditing: Avoids requesting the notification permission on page load +1ms
LH:status Auditing: Allows users to paste into input fields +3ms
LH:status Auditing: Use HTTP/2 +1ms
LH:status Auditing: Uses passive listeners to improve scrolling performance +8ms
LH:status Auditing: Document has a meta description +1ms
LH:status Auditing: Page has successful HTTP status code +1ms
LH:status Auditing: Document uses legible font sizes +1ms
LH:status Auditing: Links have descriptive text +1ms
LH:status Auditing: Links are crawlable +2ms
LH:status Auditing: Page isn’t blocked from indexing +2ms
LH:status Auditing: robots.txt is valid +2ms
LH:status Auditing: Tap targets are sized appropriately +2ms
LH:status Auditing: Document has a valid `hreflang` +1ms
LH:status Auditing: Document avoids plugins +1ms
LH:status Auditing: Document has a valid `rel=canonical` +1ms
LH:status Auditing: Structured data is valid +2ms
LH:status Auditing: Page didn't prevent back/forward cache restoration +0ms
LH:status Generating results... +2ms
LH:Printer html output written to /Users/asraine/lighthouse/invisibleink.asia_2024-01-10_10-15-35.report.html +48ms
LH:ChromeLauncher Killing Chrome instance 58264 +3ms
```

It's worth noting that this only happens on the old headless Chrome. Normal Chrome and new headless finish in a timely manner.

**Comment by adamraine** [comment]
So I think we can merge into #8557. I noticed the fancy 3D rendering in the page which PageSpeed has difficulty rendering because it doesn't use GPU acceleration. Looking at the [trace](https://trace.cafe/t/oGJXr55Jnb) Lighthouse collects, the main thread is indeed being hogged by tasks that would otherwise benefit from being offloaded onto a GPU. This forces the page load wait condition to time out and slows down many of artifact gatherers.

Also, I was wondering why the mobile report is produced on PSI but then I noticed the 82 seconds (almost 1.5 minutes) of blocking time.
